plugins/inputs/sqlserver/README.md

```diff
@@ -341,7 +316,7 @@ the new format. The version 2 queries only report raw metrics, no math has
 been done to calculate deltas. To graph this data you must calculate deltas
 in your dashboarding software.
 
-### Version 1 (query_version=1): This is Deprecated in 1.6, all future development will be under configuration option database_type
+### Version 1 (query_version=1): This is Deprecated in 1.16, all future development will be under configuration option database_type
```

Comment:
Why do you change the deprecated version?

Reply:
I just fixed the docs to what the code is saying.